tests/app/test_model.py

```diff
@@ -71,6 +204,18 @@ def test_framework_is_live_only_returns_live_frameworks(self):
             assert_equal(services.count(), 3)
             assert(all(s.framework.status == 'live' for s in services))
 
+    def test_lot_must_be_associated_to_the_framework(self):
+        with self.app.app_context():
+            self.setup_dummy_suppliers(1)
+            self.setup_dummy_service(
```

Comment:
It's not that clear from the setup here how this is testing for a lot associated to the framework - a comment to explain would be useful.